lib/commons/math/get-offset.js

```diff
@@ -1,91 +1,61 @@
+import { getTargetSize } from '../dom';
+
 /**
  * Get the offset between node A and node B
  * @method getOffset
  * @memberof axe.commons.math
  * @param {VirtualNode} vNodeA
  * @param {VirtualNode} vNodeB
+ * @param {Number} radius
  * @returns {number}
  */
-export default function getOffset(vNodeA, vNodeB) {
-  const rectA = vNodeA.boundingClientRect;
-  const rectB = vNodeB.boundingClientRect;
-  const pointA = getFarthestPoint(rectA, rectB);
-  const pointB = getClosestPoint(pointA, rectA, rectB);
-  return pointDistance(pointA, pointB);
-}
-
-/**
- * Get a point on rectA that is farthest away from rectB
- * @param {Rect} rectA
- * @param {Rect} rectB
- * @returns {Point}
- */
-function getFarthestPoint(rectA, rectB) {
-  const dimensionProps = [
-    ['x', 'left', 'right', 'width'],
-    ['y', 'top', 'bottom', 'height']
-  ];
-  const farthestPoint = {};
-  dimensionProps.forEach(([axis, start, end, diameter]) => {
-    if (rectB[start] < rectA[start] && rectB[end] > rectA[end]) {
-      farthestPoint[axis] = rectA[start] + rectA[diameter] / 2; // center | middle
-      return;
-    }
-    // Work out which edge of A is farthest away from the center of B
-    const centerB = rectB[start] + rectB[diameter] / 2;
-    const startDistance = Math.abs(centerB - rectA[start]);
-    const endDistance = Math.abs(centerB - rectA[end]);
-    if (startDistance >= endDistance) {
-      farthestPoint[axis] = rectA[start]; // left | top
-    } else {
-      farthestPoint[axis] = rectA[end]; // right | bottom
-    }
-  });
-  return farthestPoint;
-}
+export default function getOffset(vNodeA, vNodeB, radius = 12) {
```

Comment:
Radius could do with a more explicit name.

```suggestion
export default function getOffset(vNodeA, vNodeB, minRadiusNeighbour = 12) {
```